runtime/app-runtime.js

```diff
@@ -59,6 +51,7 @@ AppRuntime.prototype.init = function init () {
     this.componentLoader.load(it)
   })
   this.descriptorLoader.load(path.join(__dirname, 'descriptor'))
+  this.phaseToBooting()
```

Comment:
`phaseToBooting` would post a flora message before flora component been properly inited.

Reply:
Moved after the component did init.